src/components/booksComponent/BooksItem.js

```diff
@@ -14,10 +16,38 @@ const BooksItem = (props) => {
     dispatch(deleteBook(bookId));
   };
   return (
-    <div>
-      <h2>{title}</h2>
-      <p>{category}</p>
-      <button onClick={handleDeleteBook} data-bookid={id} type="button">Delete</button>
+    <div className="item-container">
+      <div className="left-item">
+        <p>{category}</p>
+        <h2>{title}</h2>
+        <span>Anonymous</span>
+        <div className="buttons">
+          <button type="button">Comments</button>
+          <span className="separation-left" />
+          <button onClick={handleDeleteBook} data-bookid={id} type="button">Remove</button>
+          <span className="separation-left" />
+          <button type="button">Edit</button>
+        </div>
+      </div>
+      <div className="progress-item">
+        <div style={{ width: '4.25rem', height: '4.25rem' }}>
+          <CircularProgressbar value={66} styles={buildStyles({ pathColor: '#0290ff' })} />
+        </div>
```

Comment:
- [ ] You have done a good job so far. However, the circular progress bar looks a little bit different from the design. Let's make it look like a real progress bar by making the radius clear. 

Your Implementation | Figma Design
:---:|:---:
![](https://i.imgur.com/EShJqck.png) | ![](https://i.imgur.com/rxnq9rZ.png)

Reply:
Oh yeah! I did not import the circular progress bar styles, you're right 👍